ipatests/test_integration/test_epn.py

```diff
@@ -344,22 +353,117 @@ def test_EPN_smoketest_3(self):
         expected_users = ["user1", "user3", "user7", "user14", "user28"]
         assert sorted(user_lst) == sorted(expected_users)
 
-    def test_EPN_nbdays(self):
+    def test_EPN_nbdays_0(self, cleanupmail):
```

Comment:
Is cleanupmail necessary here or is it to prevent cascading failures?

Reply:
It is necessary because if we add another test before it that generates emails we need to make sure /var/mail is empty before this starts.